numpy/_core/defchararray.py

```diff
@@ -18,13 +18,11 @@
 import functools
 
 from .._utils import set_module
-from .numerictypes import (
-    bytes_, str_, integer, int_, object_, character)
-from .numeric import ndarray, array as narray
-from numpy._core.multiarray import _vec_string, compare_chararrays
+from .numerictypes import bytes_, str_, character
+from .numeric import ndarray, array as narray, asarray as asnarray
+from numpy._core.multiarray import compare_chararrays
 from numpy._core import overrides
-from numpy._utils import asbytes
-import numpy
+from numpy.strings import *
```

Comment:
Note to other reviewers, we do use `import *` heavily in the private API and using it here is fine.